packages/fxa-settings/src/pages/Signin/Signin/index.tsx

```diff
@@ -0,0 +1,201 @@
+  // The avatar mixin is complex enough that I think it deserves its own ticket.
+
+  // TODO: The user-card mixin does some fancy footwork listening to see if you access token changes
+  // because it's possible for the access token to be invalidated when we display the avatar image.
+  // if we get the notification of a change, we re-render the card.
+
+  const onSubmit = useCallback(async () => {
+    try {
+      // TODO: add logic to set a new password
+      // TODO: add message in Alert Bar re: success, then navigate to where appropriate
+    } catch (e) {
+      // TODO: metrics event for error
+      // TODO: Add in localized Alert Bar message for error
+      setError(e);
+    }
+  }, []);
+
+  const [error, setError] = useState('');
+  const [password, setPassword] = useState('');
+  const useLoggedIn = () => {};
+
+  const signInWithPassword = () => {};
+
+  // TODO:
+  // Add in the Alert Bar in place of the original `success` and `error` display divs
+
+  return (
+    <>
+      <div className="mb-4">
+        {isPasswordNeeded ? (
+          <FtlMsg id="signin-password-needed-header">
+            <h1 className="card-header">
+              Enter your password{' '}
+              <span className="card-subheader">for your Firefox account</span>
+            </h1>
+          </FtlMsg>
+        ) : (
+          <>
+            <FtlMsg id="signin">
+              <h1 className="card-header">Sign in</h1>
+            </FtlMsg>
+            <>
+              {serviceLogo ? (
+                <span className="card-subheader">
+                  <FtlMsg id="signin-subheader-with-logo">Continue to </FtlMsg>
+                  {serviceLogo}
+                </span>
+              ) : (
+                <FtlMsg
+                  id="signin-subheader-without-logo"
+                  vars={{ serviceName }}
+                >
+                  <span className="card-subheader">
+                    Continue to {serviceName}
+                  </span>
+                </FtlMsg>
+              )}
+            </>
+          </>
+        )}
+      </div>
+      <section>
+        {/* Alerts and success messages originally went here */}
+        <div className="mt-9">
+          <div className="avatar-wrapper mx-auto h-24 w-24 mobileLandscape:h-40 mobileLandscape:w-40"></div>
+          <div className="my-5 text-base break-all">{email}</div>
+        </div>
+        <form noValidate {...{ onSubmit }}>
+          <input type="email" className="email hidden" value={email} disabled />
+
+          {isPasswordNeeded && (
+            <InputPassword
+              className="mb-2"
+              label={localizedPasswordFormLabel}
+              hasErrors={error.length > 0}
+              errorText={error}
+              tooltipPosition="bottom"
+              required={true}
+              autoFocus={true}
+              onChange={(e) => setPassword(e.currentTarget.value)}
```

Comment:
Also adding `anchorStart` will anchor the tooltip to the start instead of the middle. This would match parity with content-server.

Reply:
Dunno if this would count as syntactic sugar, but `anchorStart={true}` could be simplified to `anchorStart` (same same for `required` and `autoFocus`)

Reply:
Yeah that's def some syntactic sugar and applies to all other `={true}`s as well in this PR. We've also got it in our [front-end styling guide](https://mozilla.github.io/ecosystem-platform/reference/style-guides/react-style-guide) in ecosystem-platform to prefer dropping explicitly setting `={true}` where we can.

Reply:
Interesting, wonder if that's https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/jsx-boolean-value.md

Possible issues currently in **main** branch are:

```sh
git grep -n "={true}" ':!*.test.*' ':!*.stories.*' | cat

packages/fxa-payments-server/src/App.tsx:33:const RouteFallback = () => <LoadingOverlay isLoading={true} />;
packages/fxa-payments-server/src/routes/Checkout/index.tsx:238:    return <LoadingOverlay isLoading={true} />;
packages/fxa-payments-server/src/routes/Checkout/index.tsx:329:          showFxaLegalFooterLinks={true}
packages/fxa-payments-server/src/routes/Checkout/index.tsx:420:                      newPaypalAgreement={true}
packages/fxa-payments-server/src/routes/Product/SubscriptionCreate/index.tsx:287:                          newPaypalAgreement={true}
packages/fxa-payments-server/src/routes/Product/SubscriptionSuccess/index.tsx:77:              readOnly={true}
packages/fxa-payments-server/src/routes/Product/index.tsx:220:    return <LoadingOverlay isLoading={true} />;
packages/fxa-payments-server/src/routes/Subscriptions/index.tsx:209:    return <LoadingOverlay isLoading={true} />;
packages/fxa-payments-server/src/routes/Subscriptions/index.tsx:276:    return <LoadingOverlay isLoading={true} />;
packages/fxa-settings/src/components/Settings/HeaderLockup/index.tsx:32:        aria-haspopup={true}
packages/fxa-settings/src/components/Settings/Page2faReplaceRecoveryCodes/index.tsx:188:          cancellable={true}
packages/fxa-settings/src/components/Settings/PageAvatar/index.tsx:229:          disableAutomaticStylesInjection={true}
packages/fxa-settings/src/components/Settings/Security/index.tsx:59:            isLevelWithRefreshButton={true}
```

Not sure about this though:
```sh
git grep -n "[a-z]=['\"]true['\"]" ':!*.test.*' ':!*.stories.*' ':!.yarn/' ':!packages/123done/' ':!*.mustache' ':!*.css' ':!*.sh' | cat

packages/fxa-payments-server/src/components/AlertBar/index.tsx:97:                aria-hidden="true"
packages/fxa-payments-server/src/components/DialogMessage/index.tsx:59:                  aria-hidden="true"
packages/fxa-settings/src/components/InputPassword/index.tsx:101:            aria-hidden="true"
packages/fxa-settings/src/components/InputPassword/index.tsx:108:            aria-hidden="true"
packages/fxa-settings/src/components/Settings/Nav/index.tsx:109:                aria-hidden="true"
packages/fxa-settings/src/components/Settings/Nav/index.tsx:125:                aria-hidden="true"
```